pandapower/grid_equivalents/rei_generation.py

```diff
@@ -169,7 +169,7 @@ def _create_net_zpbn(net, boundary_buses, all_internal_buses, all_external_buses
     net_internal, net_external = _get_internal_and_external_nets(
             net, boundary_buses, all_internal_buses, all_external_buses,
             show_computing_time, calc_volt_angles=calc_volt_angles, runpp_fct=runpp_fct)
-    net_zpbn = net_external
+    net_zpbn = deepcopy(net_external)
```

Comment:
Every deepcopy takes time. why do you add a deepcopy here? It is already deepcopied in _get_internal_and_external_nets() line 485. Please try another option.
If you really need to deepcopy, then deepcopy `net[elm]` at line 273. It only deepcopies the relevant element tables.

Reply:
modified and pushed, please review